ush/config.sh.RRFS_AK_dev1

```diff
@@ -0,0 +1,74 @@
+MACHINE="jet"
```

Comment:
This file should not be included in this branch.

Reply:
deleted